e2e/bitrise.yml

```diff
@@ -11,68 +11,12 @@ workflows:
     - TEST_PLAN: ""
     - DESTINATION: platform=iOS Simulator,name=Bitrise iOS default,OS=latest
     - LOG_FORMATTER: xcbeautify
-    - RETRY_ON_FAILURE: "no"
     - EXPECT_TEST_FAILURE: "false"
     - CACHE_LEVEL: swift_packages
     after_run:
     - _run
     - _check_outputs
 
-  test_rosetta_simulator:
```

Comment:
This Rosetta simulator thing has always been an undocumented hack and it seems be broken now on the latest Xcodes. I remember the last time it was working was at Xcode 14.3.

Let's remove this test, it's not that we can continue supporting this feature, it's not worth testing IMHO.